crates/net/network/src/network.rs

```diff
@@ -431,7 +440,7 @@ pub(crate) enum NetworkHandleMessage {
     /// Disconnects a connection to a peer if it exists, optionally providing a disconnect reason.
     DisconnectPeer(PeerId, Option<DisconnectReason>),
     /// Adds a new listener for `NetworkEvent`.
-    EventListener(UnboundedSender<NetworkEvent>),
+    EventListener(Sender<NetworkEvent>),
```

Comment:
there are many sender and receiver libraries, so it's common to just bring the module into scope in import prelude and use sender/receiver types like this
```suggestion
    EventListener(broadcast::Sender<NetworkEvent>),
```